connect/api/v1/internal/intelligence/intelligence_rest_client.py

```diff
@@ -1,24 +1,103 @@
+            params={"user_email": user_email}
+        )
+
+        return response.data
+
+    def get_user_organization_permission_role(self, user_email, organization_id):
+        response = requests.get(
+            url=f"{self.base_url}v2/internal/user/permission/",
+            headers=self.authentication_instance.get_headers(),
+            params={"user_email": user_email, "organization_id": organization_id}
+        )
+        return response.data.role
+
+    def create_organization(self, user_email, organization_name):
+        response = requests.post(
+            url=f"{self.base_url}v2/internal//organization/",
+            headers=self.authentication_instance.get_headers(),
+            json=json.dumps({"user_email": user_email, "organization_name": organization_name})
+        )
+        return response.data
 
-        request = requests.get(
-            url=f"{self.base_url}/v2/internal/list-organizations",
+    def delete_organization(self, organization_id, user_email):
+        response = requests.delete(
+            url=f"{self.base_url}v2/internal/{organization_id}/delete-organization/",
             headers=self.authentication_instance.get_headers(),
             params={"user_email": user_email}
         )
+        return response
+
+    def update_organization(self, organization_id, organization_name):
+        response = requests.put(
+            url=f"{self.base_url}v2/internal/{organization_id}/organization/",
+            headers=self.authentication_instance.get_headers(),
+            params={"organization_name": organization_name}
+        )
+        return response
 
-        print(request)
+    def update_user_permission_organization(
+        self, organization_id, user_email, permission
+    ):
+        response = requests.put(
+            url=f"{self.base_url}v2/internal/user/permission/",
+            headers=self.authentication_instance.get_headers(),
+            params={"organization_id": organization_id, "user_email": user_email},
+            json=json.dumps({"role": permission})
+        )
+        return response.data
+
+    def get_organization_intelligences(self, intelligence_name, organization_id):
+
+        response = requests.get(
+            url=f"{self.base_url}v2/internal/repository/",
+            headers=self.authentication_instance.get_headers(),
+            params={"intelligence_name": intelligence_name, "org_id": organization_id}
+        )
+
+        return response.data
+
+    def update_language(self, user_email, language):
+        response = requests.put(
+            url=f"{self.base_url}v2/internal/user/language/",
+            headers=self.authentication_instance.get_headers(),
+            params={"user_email": user_email},
+            json=json.dumps({"language": language})
+        )
+        return response.data
+
+    def get_organization_statistics(self, organization_id):
+        response = requests.get(
+            url=f"{self.base_url}v2/internal/repository/",
```

Comment:
Shouldn't this endpoint be `/v2/internal/organization/{organization_id}`? It is  the only one that returns `repositories_count`